UNRELEASED.md

```diff
@@ -0,0 +1,4 @@
+**Fixed:**
+
+- bpk-component-rating:
+   - Support rating `value` when type is `string`.
```

Comment:
```suggestion
   - Support rating `value` when type is a localized `string` number.
   - Fixed `RATING_SIZES.large` not aligning with baseline
   - Update the value of `RATING_SIZES.large` from `lg` to `large` to avoid propTypes check warning 
```

Reply:
Thanks your suggestions, i've update the `UNRELEASED.md`